.github/workflows/conformance.yaml

```diff
@@ -0,0 +1,52 @@
+name: Java Conformance CI
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [
+          11.x
+          # 12.x,
+          # 13.x
+        ]
+    steps:
+    - uses: actions/checkout@v2
+    
+    - name: Set up JDK ${{ matrix.java }}
+      uses: actions/setup-java@v1
+      with:
+        java-version: ${{ matrix.java }}
+    
+    - name: Setup Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: '1.15'
+    
+    - name: Build API with Maven
+      run: (cd functions-framework-api/ && mvn install)
+
+    - name: Build invoker with Maven
+      run: (cd invoker/ && mvn install)
+    
+    - name: Run HTTP conformance tests
+      uses: GoogleCloudPlatform/functions-framework-conformance/action@v0.3.7
+      with:
+        functionType: 'http'
+        useBuildpacks: false
+        validateMapping: false
+        cmd: "'mvn -f invoker/conformance/pom.xml function:run -Drun.functionTarget=com.google.cloud.functions.conformance.HttpConformanceFunction'"
+        startDelay: 10
```

Comment:
Specifying `startDelay` should only be necessary when using buildpacks. The other frameworks that give a command directly don't give it. Unless you found it necessary I expect it can be removed.

Reply:
I found it was necessary because it takes a few seconds for the `mvn function:run` command to spin up the local server

Reply:
Maybe there is a better way for me to run the function? I don't have much experience with maven.

Reply:
Ok cool, that's fine then. I don't either to be honest.

Reply:
FWIW I'm pretty sure that's what I used when I was invoking the conformance tests manually last year. LGTM.